youtube_dl/extractor/fivethirtyeight.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from youtube_dl.compat import compat_urllib_request
+from youtube_dl import utils
+
+
+class FivethirtyeightIE(InfoExtractor):
+    _VALID_URL = r'http://fivethirtyeight\.com/.+'
+    _TEST = {
+        'url': 'http://fivethirtyeight.com/features/rage-against-the-machines/',
+        'md5': 'c825a057981316c4d4444fefea35a108',
+        'info_dict': {
+            'id': '11694550',
+            'ext': 'mp4',
+            'title': 'Rage Against The Machines',
+            'description': 'This is an excerpted chapter from “The Signal and the Noise: Why So Many Predictions Fail — but Some Don’t” by Nate Silver, editor in chief of FiveThirtyEight. …',
+            'duration': 1037,
+        }
+    }
+
+    def _real_extract(self, url):
+        webpage = self._download_webpage(url, 'video_id')
+        video_id = self._html_search_regex(r'.*data-video-id=\'(.*)\' data-cms.*', webpage, 'video_id')
```

Comment:
`.*` on both ends of regex make no sense. Also when capturing at least one character is required - there is no sense capturing empty video id.